ext/sockets/sockets.c

```diff
@@ -1338,10 +1340,9 @@ PHP_FUNCTION(socket_bind)
 				sa->sun_family = AF_UNIX;
 
 				if (addr_len >= sizeof(sa->sun_path)) {
-					php_error_docref(NULL, E_WARNING,
-							"Invalid path: too long (maximum size is %d)",
-							(int)sizeof(sa->sun_path) - 1);
-					RETURN_FALSE;
+					zend_argument_value_error(2, "is too long (maximum size is %d)",
```

Comment:
I think either the previous similar message should contain `(maximum size is %d)`, or neither of them. Another possibility is to in both cases write `must have a size less than %d` (I don't how to correctly phrase this, so please fix it :) )